pkg/prom/cluster/config.go

```diff
@@ -0,0 +1,49 @@
+package cluster
+
+import (
+	"flag"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/ring"
+	"github.com/cortexproject/cortex/pkg/ring/kv"
+	"github.com/grafana/agent/pkg/prom/ha/client"
+	flagutil "github.com/grafana/agent/pkg/util"
+)
+
+// DefaultConfig provides default values for the config
+var DefaultConfig = *flagutil.DefaultConfigFromFlags(&Config{}).(*Config)
+
+// Config describes how to instantiate a scraping service Server instance.
+type Config struct {
+	Enabled         bool                  `yaml:"enabled"`
+	ReshardInterval time.Duration         `yaml:"reshard_interval"`
+	ReshardTimeout  time.Duration         `yaml:"reshard_timeout"`
+	Client          client.Config         `yaml:"client"`
```

Comment:
I moved the client config here, which will be a change that I need to handle when this is wired in.